sdk/Sdk/worker.config.json

```diff
@@ -3,6 +3,7 @@
     "language": "dotnet-isolated",
     "extensions": [ ".dll" ],
     "defaultExecutablePath": "$functionExe$",
-    "defaultWorkerPath": "$functionWorker$"
+    "defaultWorkerPath": "$functionWorker$",
+    "workerIndexing":  "true"
```

Comment:
Are we still considering moving this to [Worker Capabilities?](https://github.com/Azure/azure-functions-dotnet-worker/blob/main/src/DotNetWorker.Grpc/GrpcWorker.cs#L237)

Reply:
No we're moving forward with worker config and a [gRPC message flag](https://github.com/Azure/azure-functions-language-worker-protobuf/blob/73658b78e6dd876af13e50488542f63aac544df6/src/proto/FunctionRpc.proto#L307) to handle legacy apps and worker indexing.